src/builtins/type_coersion.md

```diff
@@ -84,6 +84,14 @@ toInt8(bool): int8
 toInt8(string): int8
 ```
 
+##### Type aliases
```

Comment:
can we add the type alias directly in the original sections so `toInt64` and line 31 should be `toInt64 or toInt` such that? and thank you :)

```
##### toInt or toInt64
```alan
// Converts the built-in basic types to 64-bit integers which are the default numeric type
toInt(int8): int         // toInt64(int8): int64
toInt(int16): int        // toInt64(int16): int64
toInt(int32): int        // toInt64(int32): int64
toInt(int): int          // toInt64(int64): int64
toInt(float32): int      // toInt64(float32): int64
toInt(float64): int      // toInt64(float64): int64
toInt(bool): int         // toInt64(bool): int64
toInt(string): int       // toInt64(string): int64
```

Reply:
Like this? :)

Reply:
Yes, exactly :)

I somehow missed this. I'm going to go ahead and land it, thank you!